java/src/com/github/bazel_contrib/contrib_rules_jvm/junit5/README.md

```diff
@@ -0,0 +1,280 @@
+# JUnit 5 Test Runner
```

Comment:
This document is amazing and it should be shortlinked from the documentations of most of the tools mentioned here. Thanks for the effort writing it!